blackbird_python/blackbird/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = '0.3.0'
+__version__ = '0.3.1-dev'
```

Comment:
Just curious: is it more frequent to have minor releases for bb? For SF and PL I'd guess they are mostly bug fix releases

Reply:
There haven't been so many releases for Blackbird; this is the first significant one in a while. Not sure if next release will be a patch release or a larger one, but I don't think it's unlikely that it will be a minor release. :thinking:

Reply:
When the major version is 0 in semver.org, the goal is rapid development. The strict rules of semver don't apply until we hit major version 1.0. This is why we typically only increment the minor number, regardless of the size of the release.

> **How should I deal with revisions in the 0.y.z initial development phase?**
>
> The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.